photon-lib/src/main/java/org/photonvision/RobotPoseEstimator.java

```diff
@@ -93,37 +97,37 @@ public RobotPoseEstimator(
      *
      * @return The updated estimated pose and the latency in milliseconds
      */
-    public Pair<Pose3d, Double> update() {
+    public Optional<Pair<Pose3d, Double>> update() {
         if (cameras.isEmpty()) {
             DriverStation.reportError("[RobotPoseEstimator] Missing any camera!", false);
-            return Pair.of(lastPose, 0.);
+            return Optional.empty();
         }
         Pair<Pose3d, Double> pair;
         switch (strategy) {
             case LOWEST_AMBIGUITY:
                 pair = lowestAmbiguityStrategy();
                 lastPose = pair.getFirst();
-                return pair;
+                return Optional.of(pair);
```

Comment:
Nit: I bet you could initialize the pair to null at the top of the switch-case, and have the default-case break. Then, you only need one `return optional.ofNullable(pair)` at the bottom.

Reply:
Was thinking that too 🙂 Figured I'd make the simplest change first. I will update that